wakeword.js

```diff
@@ -324,8 +305,8 @@ module.exports = {
                 break;
 
             case stateEnum.STREAMING:
-                if (this.detected && this.detected.length > 0) {
-                    this.decoder.startUtt();
+                if (this.detected /*&& this.detected.length > 0*/) {
+                    //this.decoder.startUtt();
```

Comment:
I know this is just a prototype but I think it's worth removing this commented-out code.